docs/paper/paper.md

```diff
@@ -25,16 +25,28 @@ bibliography: paper.bib
 # Summary
 
 CaPS is a package for the analysis of the Casimir effect in the plane-sphere
-geometry. This geometry depicted in Fig. 1 is most commonly used in precision
-measurements of the Casimir force arising from quantum and thermal fluctuations
-of the electromagnetic field [@bordag_advances]. Specifically, CaPS allows to
-compute the Casimir free energy as a function of the sphere radius $R$, the
-minimal separation $L$ between sphere and plane, the temperature $T$, and the
-material properties of plane and sphere. It is assumed that both objects are
-non-magnetic and placed in vacuum. In typical experiments the aspect ratio
-$R/L$ is of the order of 1000. The main purpose of this package is to
-make aspect ratios as large as $R/L\sim5000$ accessible. Higher aspect ratios
-are usually well covered by the proximity force approximation.
+geometry. The Casimir force arises due to quantum and thermal fluctuations of
+the electromagnetic field and is closely related to the van der Waals forces
+[@bordag_advances]. As an observable consequence of vacuum fluctuations in the
+macroscopic world, the Casimir effect is linked to the problem of zero-point
+energy and the cosmological constant problem. The Casimir force is the dominant
+force between neutral non-magnetic materials in the nanometre to micrometre
+range. As such, a precise knowledge of the Casimir force is crucial in order to
+detect possible deviations from the gravitational interaction, and thus to
+either exclude or possibly support proposed mechanisms a fifth fundamental
+interaction. From a technological point of view, the Casimir effect has been
+shown to play an important role for micro- and nano-electromechanical systems.
```

Comment:
I would propose to shift the emphasis somewhat:

The Casimir force arises due to quantum and thermal fluctuations of the electromagnetic field and is closely related to the van der Waals force [@bordag_advances]. It is the dominant force between neutral non-magnetic materials in the nanometre to micrometre range and plays an important role in colloidal systems. Of technological relevance are applications to micro- and nano-electromechanical systems where the Casimir force can lead to stiction and thus constitute a failure mechanism. On a more fundamental level, the Casimir effect is linked to the zero-point energy and the cosmological constant problem. A precise knowledge of the Casimir force is crucial for the search for possible deviations from Newton's law of gravitation which could arise from a fifth fundamental interaction.

Reply:
I am happy with the change. There is just one (minor) thing that I don't like right now: The Casimir force being the dominant force in nm and µm range and the 5th fundamental interaction are separated now.

Reply:
Length scales are also of relevance for the applications mentioned before. A possible solution could be to recall it: ... for possible deviations from Newton's law of gravitation on the nanometre scale which could ...

Reply:
I would not repeat it as there are only two sentences inbetween and it is not crucial for the understanding of thre reader. I would keep it as it is because I like that we group the "more fundamental" applications.

Reply:
Do you want to put a reference on the latter point?

Reply:
I have not added any references to the introduction yet.

It might be good to have a reference for the MEMS/NEMS, cosmological constant problem, and the 5th force. What do you think?

Reply:
Probably we should stick to one reference per point. For the last point, I would propose Ref. 13 of your thesis. For the other two points, I leave the choice to you (probably 16 and one of 17-19).

Reply:
Okay, I will add the references to the PR tomorrow.

Reply:
Ok. Feel free to merge after adding the references (even if you decide to add two references on MEMS and NEMS).